app/src/main/java/com/ckarthickit/mealdb/viewmodel/MealCatalogViewModel.kt

```diff
@@ -52,4 +53,8 @@ class MealCatalogViewModel : androidx.lifecycle.ViewModel() {
     // cancel all
     backgroundWorkScope.cancel()
   }
+
+  fun emptyFun(){
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ckarthickit2/android-best-practices/blob/cedc04382b578b60decf268a45205dd0f299fa43/app/src/main/java/com/ckarthickit/mealdb/viewmodel/MealCatalogViewModel.kt#L57"></span>This empty block of code can be removed.</td>
    </tr>
  </tbody>
</table>